stdlib/SparseArrays/test/higherorderfns.jl

```diff
@@ -27,6 +27,9 @@ include("forbidproperties.jl")
         @test map!(cos, X, A) == sparse(map!(cos, fX, fA))
         @test_throws DimensionMismatch map!(sin, X, spzeros((shapeA .- 1)...))
     end
+    # https://github.com/JuliaLang/julia/issues/37819
+    Z = spzeros(50000, 50000)
+    -Z
```

Comment:
Maybe turn this into a little test?
```suggestion
    @test isa(-Z, SparseMatrixCSC{<:Any,Int64})
```
Same below.

Reply:
Done.